conans/client/build/cmake_flags.py

```diff
@@ -280,7 +282,7 @@ def _get_make_program_definition(self):
 
         return {}
 
-    def get_definitions(self):
+    def get_definitions(self, cmake_version=None):
```

Comment:
Why a default value here? This function is called only from this line

```
self.definitions = builder.get_definitions(cmake_version=self.get_version())
```

and it will always pass a value or raise.

Reply:
It was because a test was calling directly and also thinking about a tool or something that could be calling this... I'll update the test and remove that.